File: RxUI/src/main/java/com/vondear/rxui/view/RxAutoCompleteTextView.java
package com.vondear.rxui.view;

import android.content.Context;
import android.util.AttributeSet;

import androidx.appcompat.widget.AppCompatAutoCompleteTextView;

/**
 * @author vondear
 * @date 2018/2/2
 * <p>
 * 解决没有输入内容时也能展示筛选内容的需求
 */

public class RxAutoCompleteTextView extends AppCompatAutoCompleteTextView {

    public RxAutoCompleteTextView(Context context) {
        super(context);
    }

    public RxAutoCompleteTextView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public RxAutoCompleteTextView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    public boolean enoughToFilter() {
        return true;
    }
}

File: RxUI/src/main/java/com/tamsiree/rxui/view/RxAutoCompleteTextView.kt
package com.tamsiree.rxui.view

import android.content.Context
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatAutoCompleteTextView

/**
 * @author tamsiree
 * @date 2018/2/2
 *
 *
 * 解决没有输入内容时也能展示筛选内容的需求
 */
class RxAutoCompleteTextView : AppCompatAutoCompleteTextView {
    constructor(context: Context?) : super(context!!)
    constructor(context: Context?, attrs: AttributeSet?) : super(context!!, attrs)
    constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(context!!, attrs, defStyleAttr)

    override fun enoughToFilter(): Boolean {
        return true
    }
}